pyocd/__main__.py

```diff
@@ -49,6 +49,8 @@
 from .flash.eraser import FlashEraser
 from .flash.file_programmer import FileProgrammer
 from .core import options
+from .utility.cmdline import split_command_line
```

Comment:
Is `split_command_line()` used anywhere?

Reply:
split_command_line is already imported in line 39, thanks!